modules/etcd-node-scaling.adoc

```diff
@@ -10,6 +10,11 @@ In general, clusters must have 3 control plane nodes. However, if your cluster i
 
 Scaling a cluster to 4 or 5 control plane nodes is available only on bare metal platforms.
 
+[NOTE]
+====
```

Comment:
Is is also possible using assisted-installer.

Reply:
I can add that to this note